common-npm-packages/azure-arm-rest-v2/azure-arm-endpoint.ts

```diff
@@ -30,6 +30,21 @@ export class AzureRMEndpoint {
             return this.endpoint;
         }
         else {
+
+            const rawUseMSAL = tl.getVariable("USE_MSAL");
+            if (rawUseMSAL) {
+                try {
+                    tl.debug(`MSAL - USE_MSAL override is found: ${rawUseMSAL}`);
+                    if (typeof JSON.parse(rawUseMSAL) !== "boolean") {
```

Comment:
We may save one call of `JSON.parse` by storing the result in variable.

Reply:
updated.